README.md

```diff
@@ -172,15 +172,15 @@ The two aforementioned requirements are not only satisfied by the Occlum toolcha
 
 Occlum is being actively developed. We now focus on implementing more system calls and additional features required in the production environment.
 
-While this project is still not mature or stable (we are halfway through reaching version 1.0.0), we have used Occlum to port many real-world applications (like Tensorflow Lite, XGBoost, GCC, Lighttpd, etc.) to SGX with little or no source code modifications. We believe that the current implementation of Occlum is already useful to many users and ready to be deployed in some use cases. 
+While this project is still not mature or stable (we are halfway through reaching version 1.0.0), we have used Occlum to port many real-world applications (like Tensorflow Lite, XGBoost, GCC, Lighttpd, etc.) to SGX with little or no source code modifications. We believe that the current implementation of Occlum is already useful to many users and ready to be deployed in some use cases.
 
 ## Why the Name?
 
-The project name Occlum stems from the word *Occlumency* coined in Harry Porter series by J. K. Rowling. In *Harry Porter and the Order of Pheonix*, Occlumency is described as:
+The project name Occlum stems from the word *Occlumency* coined in Harry Potter series by J. K. Rowling. In *Harry Potter and the Order of Phoenix*, Occlumency is described as:
 
 > The magical defence of the mind against external penetration. An obscure branch of magic, but a highly useful one... Used properly, the power of Occlumency will help shield you from access or influence.
 
-The same thing can be said to Occlum, not for mind, but program:
+The same thing can be said for Occlum, not the for mind, but for the program:
```

Comment:
I think "the" should be put after "for" right before "mind".

Reply:
Yes. I added that.